src/app/components/AnalyticsConsent/redact.ts

```diff
@@ -0,0 +1,15 @@
+export function redactURL(u: string) {
```

Comment:
This looks really great, but are we sure we want redacted urls? Don't see a reason, unless the reason is searching tied to IP address?

It does add a benefit of easily aggregating pages that were visited though.

Reply:
Yeah, unlinkability of multiple accounts a single user is often visiting (not just ip in this case: unique visitor id)